test/fika/erl_translate_test.exs

```diff
@@ -67,6 +67,15 @@ defmodule Fika.ErlTranslateTest do
     assert result == forms
   end
 
+  test "logical operators" do
+    str = "(true || !false) && true"
+
+    ast = Parser.expression!(str)
+
+    assert {:op, 1, :and, {:op, 1, :or, {:atom, 1, true}, {:op, 1, :not, {:atom, 1, false}}},
+            {:atom, 1, true}} = ErlTranslate.translate_expression(ast)
+  end
```

Comment:
I'm curious on what happens if we use `(:true || :false)`, for instance

Reply:
As you can see in this test, in the end :boolean is cast to :atom, so I think that if something can go wrong, it will be in the type checker

Reply:
yeah, at erlang level booleans don't exist